xstate-chatbot/nodejs/src/machine/hospital-details.js

```diff
@@ -0,0 +1,1068 @@
+              message += `\t::\t ${context.slots.previoushospitaldata.discharged_covid_patients_l2}`;
+              message += `\n`;
+              message += dialog.get_message(messages.confirmedPreviousCOVIDPatientsWithoutOxygen.prompt, context.user.locale);
+              message += `\t::\t ${context.slots.previoushospitaldata.confirmed_cases_on_oxygen_without_support_l2}`;
+              message += `\n`;
+              dialog.sendMessage(context, message);
+            }),
+            target: '#endstate',
+          },
+        ],
+
+      },
+    },//update L2 existing Details of hospital
+    l2ConfirmedCOVIDPatientsOnOxygen: {
+      id: 'l2ConfirmedCOVIDPatientsOnOxygen',
+      initial: 'prompt',
+      states: {
+        prompt: {
+          onEntry: assign((context, event) => {
+            let message = dialog.get_message(messages.confirmedPreviousCOVIDPatientsOnOxygen.prompt, context.user.locale);
+            message += `\n :: ${context.slots.previoushospitaldata.confirmed_cases_on_oxygen_support_l2}`;
+            message += `\n`;
+            message += dialog.get_message(messages.confirmedCOVIDPatientsOnOxygen.prompt, context.user.locale);
+            dialog.sendMessage(context, message);
+          }),
+          on: {
+            USER_MESSAGE: 'process',
+          },
+        },
+        process: {
+          onEntry: assign((context, event) => {
+            const message = dialog.get_input(event, false);
+            if (event.message.type == 'text') {
+              const message = dialog.get_input(event, false);
+              context.slots.hospital.confirmed_cases_on_oxygen_support_l2 = message;
+              context.validMessage = true;
+            } else {
+              context.validMessage = false;
+            }
+          }),
+          always: [
+            {
+              cond: (context) => context.validMessage == false,
+              target: 'error',
+            },
+            {
+              cond: (context) => context.validMessage == true,
+              target: '#l2ConfirmedCOVIDPatientsWithoutOxygen',
+            },
+          ],
+        },
+        error: {
+          onEntry: assign((context, event) => {
+            dialog.sendMessage(context, dialog.get_message(messages.confirmedCOVIDPatientsOnOxygen.error, context.user.locale), false);
+          }),
+          always: 'prompt',
+        },
+      },
+    },//Number of confirmed COVID Patients on oxygen Oxygen 
+    l2ConfirmedCOVIDPatientsWithoutOxygen: {
+      id: 'l2ConfirmedCOVIDPatientsWithoutOxygen',
+      initial: 'prompt',
+      states: {
+        prompt: {
+          onEntry: assign((context, event) => {
+            let message = dialog.get_message(messages.confirmedPreviousCOVIDPatientsWithoutOxygen.prompt, context.user.locale);
+            message += `\t::\t ${context.slots.previoushospitaldata.confirmed_cases_on_oxygen_without_support_l2}`;
+            message += `\n`;
+            message += `\n`;
```

Comment:
Merge both \n with the previous line

Reply:
removed one .One '\n' is sufficient

Reply:
This \n could also go at the end of the previous line. Doesn't make much of a difference. Your choice.

Reply:
rearranged